packages/devtools-reps/src/reps/string.js

```diff
@@ -75,6 +92,47 @@ function StringRep(props) {
     ...getLinkifiedElements(text, shouldCrop && cropLimit, omitLinkHref, openLink));
```

Comment:
Now i'm not sure this would be handled like it should for long string.
Here we assume the string contains at least one URL, and we render them as actual URL's.

We should have new tests for longStrings to ensure this case is handled correctly.